Could not find property 'translateClassPaths' on com.github.j2objccontrib.j2objcgradle.J2objcConfig_Decorated@5108b90b.

Using the latest 0.3.0-alpha plugin master I get the following error.

Here is my build.gradle config: 

```
j2objcConfig {
  dependsOnJ2objcLib ":majestella-base"
  
  generatedSourceDirs "build/source/apt"  

  xcodeProjectDir "${projectDir}/../Majestella-IOS"
  xcodeTarget "Majestella"
  
  translateArgs "--no-package-directories"
  translateArgs "--prefixes", "prefixes.properties"
  translateArgs "-use-arc"
  translateArgs "--doc-comments"
  translateArgs "--build-closure"
  translateArgs "--generate-deprecated"
  
  translateClassPaths "lib/json-20140107.jar", "lib/javax.inject-1.jar", "lib/auto-factory-1.0-beta2.jar", "lib/auto-service-1.0-rc2.jar", "lib/javawriter-2.5.1.jar", "lib/guava-18.0.jar"
  translateSourcepaths "${projectDir}/libSrc/dagger-2.0.1-sources.jar"
  
 
  extraObjcCompilerArgs += '-fobjc-arc'
  
//  translateJ2objcLibs = [ "j2objc_annotations.jar", "j2objc_guava.jar", "javax.inject-1.jar", "jsr305-3.0.0.jar" ]
   
  testMinExpectedTests 0
  filenameCollisionCheck = true
  cycleFinderExpectedCycles 0

  cycleFinderArgs "--whitelist", "/Applications/J2Objc/j2objc-sorce/jre_emul/cycle_whitelist.txt"
  cycleFinderArgs "--sourcefilelist", "/Applications/J2Objc/j2objc-sorce/jre_emul/build_result/java_sources.mf"
  
  
  finalConfigure()
}
```
The error is: 

```
gradle clean --stacktrace
Project 'majestella-main' is missing finalConfigure():
https://github.com/j2objc-contrib/j2objc-gradle/blob/master/FAQ.md#How-do-I-call-finalConfigure

FAILURE: Build failed with an exception.

* Where:
Build file '/Users/mg/Documents/Grails/GGTS3.6.3-SR1-2/majestella-main/build.gradle' line: 93

* What went wrong:
A problem occurred evaluating root project 'majestella-main'.
> Could not find property 'translateClassPaths' on com.github.j2objccontrib.j2objcgradle.J2objcConfig_Decorated@5108b90b.

* Try:
Run with --info or --debug option to get more log output.

* Exception is:
org.gradle.api.GradleScriptException: A problem occurred evaluating root project 'majestella-main'.
	at org.gradle.groovy.scripts.internal.DefaultScriptRunnerFactory$ScriptRunnerImpl.run(DefaultScriptRunnerFactory.java:76)
	at org.gradle.configuration.DefaultScriptPluginFactory$ScriptPluginImpl$1.run(DefaultScriptPluginFactory.java:148)
	at org.gradle.configuration.DefaultScriptPluginFactory$ScriptPluginImpl.apply(DefaultScriptPluginFactory.java:156)
	at org.gradle.configuration.project.BuildScriptProcessor.execute(BuildScriptProcessor.java:39)
	at org.gradle.configuration.project.BuildScriptProcessor.execute(BuildScriptProcessor.java:26)
	at org.gradle.configuration.project.ConfigureActionsProjectEvaluator.evaluate(ConfigureActionsProjectEvaluator.java:34)
	at org.gradle.configuration.project.LifecycleProjectEvaluator.evaluate(LifecycleProjectEvaluator.java:55)
	at org.gradle.api.internal.project.AbstractProject.evaluate(AbstractProject.java:487)
	at org.gradle.api.internal.project.AbstractProject.evaluate(AbstractProject.java:85)
	at org.gradle.execution.TaskPathProjectEvaluator.configureHierarchy(TaskPathProjectEvaluator.java:42)
	at org.gradle.configuration.DefaultBuildConfigurer.configure(DefaultBuildConfigurer.java:35)
	at org.gradle.initialization.DefaultGradleLauncher.doBuildStages(DefaultGradleLauncher.java:129)
	at org.gradle.initialization.DefaultGradleLauncher.doBuild(DefaultGradleLauncher.java:106)
	at org.gradle.initialization.DefaultGradleLauncher.run(DefaultGradleLauncher.java:86)
	at org.gradle.launcher.exec.InProcessBuildActionExecuter$DefaultBuildController.run(InProcessBuildActionExecuter.java:90)
	at org.gradle.tooling.internal.provider.ExecuteBuildActionRunner.run(ExecuteBuildActionRunner.java:28)
	at org.gradle.launcher.exec.ChainingBuildActionRunner.run(ChainingBuildActionRunner.java:35)
	at org.gradle.launcher.exec.InProcessBuildActionExecuter.execute(InProcessBuildActionExecuter.java:41)
	at org.gradle.launcher.exec.InProcessBuildActionExecuter.execute(InProcessBuildActionExecuter.java:28)
	at org.gradle.launcher.exec.DaemonUsageSuggestingBuildActionExecuter.execute(DaemonUsageSuggestingBuildActionExecuter.java:50)
	at org.gradle.launcher.exec.DaemonUsageSuggestingBuildActionExecuter.execute(DaemonUsageSuggestingBuildActionExecuter.java:27)
	at org.gradle.launcher.cli.RunBuildAction.run(RunBuildAction.java:40)
	at org.gradle.internal.Actions$RunnableActionAdapter.execute(Actions.java:169)
	at org.gradle.launcher.cli.CommandLineActionFactory$ParseAndBuildAction.execute(CommandLineActionFactory.java:237)
	at org.gradle.launcher.cli.CommandLineActionFactory$ParseAndBuildAction.execute(CommandLineActionFactory.java:210)
	at org.gradle.launcher.cli.JavaRuntimeValidationAction.execute(JavaRuntimeValidationAction.java:35)
	at org.gradle.launcher.cli.JavaRuntimeValidationAction.execute(JavaRuntimeValidationAction.java:24)
	at org.gradle.launcher.cli.CommandLineActionFactory$WithLogging.execute(CommandLineActionFactory.java:206)
	at org.gradle.launcher.cli.CommandLineActionFactory$WithLogging.execute(CommandLineActionFactory.java:169)
	at org.gradle.launcher.cli.ExceptionReportingAction.execute(ExceptionReportingAction.java:33)
	at org.gradle.launcher.cli.ExceptionReportingAction.execute(ExceptionReportingAction.java:22)
	at org.gradle.launcher.Main.doAction(Main.java:33)
	at org.gradle.launcher.bootstrap.EntryPoint.run(EntryPoint.java:45)
	at org.gradle.launcher.bootstrap.ProcessBootstrap.runNoExit(ProcessBootstrap.java:54)
	at org.gradle.launcher.bootstrap.ProcessBootstrap.run(ProcessBootstrap.java:35)
	at org.gradle.launcher.GradleMain.main(GradleMain.java:23)
Caused by: groovy.lang.MissingPropertyException: Could not find property 'translateClassPaths' on com.github.j2objccontrib.j2objcgradle.J2objcConfig_Decorated@5108b90b.
	at org.gradle.api.internal.AbstractDynamicObject.propertyMissingException(AbstractDynamicObject.java:43)
	at org.gradle.api.internal.AbstractDynamicObject.getProperty(AbstractDynamicObject.java:35)
	at org.gradle.api.internal.CompositeDynamicObject.getProperty(CompositeDynamicObject.java:97)
	at com.github.j2objccontrib.j2objcgradle.J2objcConfig_Decorated.getProperty(Unknown Source)
	at com.github.j2objccontrib.j2objcgradle.J2objcConfig$_dependsOnJ2objcLib_closure1_closure4.doCall(J2objcConfig.groovy:289)
	at org.gradle.api.internal.ClosureBackedAction.execute(ClosureBackedAction.java:67)
	at org.gradle.api.internal.plugins.ExtensionsStorage$ExtensionHolder.configure(ExtensionsStorage.java:145)
	at org.gradle.api.internal.plugins.ExtensionsStorage.configureExtension(ExtensionsStorage.java:69)
	at org.gradle.api.internal.plugins.DefaultConvention$ExtensionsDynamicObject.invokeMethod(DefaultConvention.java:207)
	at org.gradle.api.internal.CompositeDynamicObject.invokeMethod(CompositeDynamicObject.java:150)
	at org.gradle.api.internal.project.DefaultProject_Decorated.invokeMethod(Unknown Source)
	at com.github.j2objccontrib.j2objcgradle.J2objcConfig$_dependsOnJ2objcLib_closure1.doCall(J2objcConfig.groovy:288)
	at com.github.j2objccontrib.j2objcgradle.J2objcConfig.dependsOnJ2objcLib(J2objcConfig.groovy:264)
	at com.github.j2objccontrib.j2objcgradle.J2objcConfig$dependsOnJ2objcLib.callCurrent(Unknown Source)
	at com.github.j2objccontrib.j2objcgradle.J2objcConfig.dependsOnJ2objcLib(J2objcConfig.groovy:245)
	at build_6jaiihyu4lnu78ywb4x7it7j5$_run_closure5.doCall(/Users/mg/Documents/Grails/GGTS3.6.3-SR1-2/majestella-main/build.gradle:93)
	at org.gradle.api.internal.ClosureBackedAction.execute(ClosureBackedAction.java:67)
	at org.gradle.api.internal.plugins.ExtensionsStorage$ExtensionHolder.configure(ExtensionsStorage.java:145)
	at org.gradle.api.internal.plugins.ExtensionsStorage.configureExtension(ExtensionsStorage.java:69)
	at org.gradle.api.internal.plugins.DefaultConvention$ExtensionsDynamicObject.invokeMethod(DefaultConvention.java:207)
	at org.gradle.api.internal.CompositeDynamicObject.invokeMethod(CompositeDynamicObject.java:150)
	at org.gradle.groovy.scripts.BasicScript.methodMissing(BasicScript.java:79)
	at build_6jaiihyu4lnu78ywb4x7it7j5.run(/Users/mg/Documents/Grails/GGTS3.6.3-SR1-2/majestella-main/build.gradle:92)
	at org.gradle.groovy.scripts.internal.DefaultScriptRunnerFactory$ScriptRunnerImpl.run(DefaultScriptRunnerFactory.java:74)
	... 35 more


BUILD FAILED
```

--- a/src/main/groovy/com/github/j2objccontrib/j2objcgradle/J2objcConfig.groovy
+++ b/src/main/groovy/com/github/j2objccontrib/j2objcgradle/J2objcConfig.groovy
@@ -286,7 +286,7 @@ class J2objcConfig {
 
             logger.debug("$project:j2objcTranslate must use ${beforeProject.jar.archivePath}")
             j2objcConfig {
-                translateClassPaths += beforeProject.jar.archivePath.absolutePath
+                translateClasspaths += beforeProject.jar.archivePath.absolutePath
             }
         }
 

